invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/SearchResultItemLayout.js

```diff
@@ -73,8 +78,7 @@ class SearchResultItemComponent extends Component {
           collapsing
           className="word-break-all"
         >
-          {/* TODO needs to be expanded */}
-          {result.parent.access.owned_by.user}
+          {(recordOwner && recordOwner.user) || "System"}
```

Comment:
please leave a comment with a link to backend fix issue so that we replace it with the long term solution :)